webapp/actions/websocket_actions.jsx

```diff
@@ -241,6 +245,12 @@ function handleUserRemovedEvent(msg) {
     }
 }
 
+function handleUserUpdatedEvent(msg) {
+    if (UserStore.getCurrentId() !== msg.user_id) {
+        AsyncClient.getProfiles();
```

Comment:
I think @enahum has a bunch of tickets to go add APIs like that and to improve scalability for systems with lots of users